pkgs/applications/audio/psst/default.nix

```diff
@@ -0,0 +1,38 @@
+{ pkgs, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "psst";
+  version = "0.1.0";
+  buildAndTestSubdir = "psst-gui";
```

Comment:
Is this cargo related?

Reply:
Yes. The repository is a crago workespace with multiple crates. The `psst-gui` subdirectory contains the actual binary project. The `buildAndTestSubdir` tells cargo which which directory to build.

https://github.com/jpochyla/psst/tree/master/psst-gui

Reply:
Lets move it below cargoSha256 to make that clear.

Reply:
Done in 9adf5fd